src/Npgsql/Replication/PhysicalReplicationConnection.cs

```diff
@@ -69,6 +71,24 @@ async Task<PhysicalReplicationSlot> CreatePhysicalReplicationSlot(string slotNam
         }
     }
 
+    /// <summary>
+    /// Read some information associated to a replication slot.
+    /// Returns a tuple with <see langword="null"/> values if the replication slot does not exist.
```

Comment:
This is out of date. I think it's probably enough to document the possible null return value in the `<returns>` docs below - probably no need to repeat here in the summary.